tests/e2e/Services/Sites/SitesBase.php

```diff
@@ -267,6 +269,27 @@ protected function deleteSite(string $siteId): mixed
         return $site;
     }
 
+    protected function createSiteDomain(string $siteId, string $subdomain = ''): string
```

Comment:
Lets rename to `setupSiteDomain`.

Create = call and return response
Setup = call and assertions for success; return only relevant value